roles/falcon_installation/README.md

```diff
@@ -52,7 +52,7 @@ This example installs the Falcon Sensor:
 ---
 - hosts: all
   roles:
-  - role: falcon_installation
+  - role: crowdstrike.falcon.falcon_installation
   vars:
     falcon_api_client_id: <Falcon_UI_OAUTH_client_id>
     falcon_api_client_secret: <Falcon_UI_OAUTH_client_secret>
```

Comment:
Wasn't there some informal naming scheme for these variables?

Reply:
There is. These were agreed to with @isimluk